releng/org.eclipse.emf.mwe2.language.sdk.dummy/pom.xml

```diff
@@ -10,5 +10,5 @@
 	<packaging>eclipse-feature</packaging>
 	<groupId>org.eclipse.emf</groupId>
 	<artifactId>org.eclipse.emf.mwe2.language.sdk</artifactId>
-	<version>2.11.2.v20200224-0816</version>
+	<version>2.11.3.v20200224-0816</version>
```

Comment:
Same here

Reply:
this is just a random timestamp to satisfy tycho.